usagov/vote-gov-drupal
# VOTE-3266 update to embedded_content v2.0

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3266

## Description

Update/migrate ckeditor5_embedded_content module version 1 to embedded_content module version 2.

## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. Ensure the changes imported without error.
2. Log in and visit http://vote-gov.lndo.site/node/69, make sure the inline alert, icons and placeholders are loading
3. visit http://vote-gov.lndo.site/accessibility and make sure the touchpoints form is loading
4. visit http://vote-gov.lndo.site/register/alaska/mail-in-form-filler and make sure the alert and icons are loading and that the replacements are getting replaced correctly.
5. create a new basic page and add a new embedded content instance in the wysiwyg and test that is renders correctly when you view the page.

## Checklist for the Developer




## Checklist for the Peer Reviewers
